### DISCUSSION TITLE
how to enable pwm-fan on radxa rock5a

### DISCUSSION THREAD

**Discussion by stevepringle** [Joshua-Riek/ubuntu-rockchip#388] [Q&A]
Hello,

Does anyone know how to enable the pwm-fan on the radxa rock5a board. I've added pam-fan as an overlay in the ubuntuEnv.txt file and lsmod shows the pwm-fan module as loaded but not used by anything. I'm obviously missing something so any guidance would be gratefully received.

**Comment by guynich** [comment]
It's probably not helpful but here's how information when using Radxa's Ubuntu release [link](https://docs.radxa.com/en/rock5/rock5a/accessories-guides/heatsink-2513) - this worked for me with two different 2-wire 5V fans (neither Radxa parts).

**Reply by stevepringle** [reply]
Thanks for that, I'll investigate further.

**Reply by guynich** [reply]
Quick test: the pwm connector on Rock 5A with v1.28 Ubuntu Server does start a 2-wire fan when the CPU temperature rises.

**Reply by stevepringle** [reply]
Thanks for that. I probably haven’t got the temperature high enough. I’ll run the stress tests and check it all looks OK at higher temperatures.
